### PR TITLE
Milestone 1. Init components

### PR DESCRIPTION
**Pull request by indigodavid**
In this milestone, the following features have been added:

- PropTypes, React Router, and React Icons installed
- Added NoMatch module to handle wrong URL
- Added Book, BookList, Books modules to display book information and progress
- Added AddBook module
- Added Categories module
- Added Navbar module and integrate routing in App module
- Added positioning styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
+- Microverse
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- My supportive family 😊
 
 ## 📝 License
```
Comment (MahmoudBakr23) [inline README.md:72]: - Please fix the MIT link for a professional README file

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @indigodavid  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done handling your pages using react routing
- Great work breaking down your app into multiple reusable components
- Nicely done using ES6 features as functional components
- I liked the idea of the 404 page
- Great PR description and commit messages
- Correct Linters
- Well done using dummy data to display books dynamically
- Nice design

**Optional**
_Check the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (indigodavid) [reply README.md:72]: @MahmoudBakr23 I added the MIT.md file, thanks for the recommendation

**Comment by indigodavid** [comment]
@MahmoudBakr23 Thanks for your thorough and kind review. I'll merge the branch now.
Best regards and happy coding!!

**Pull request merged**